src/content/docs/2/guide/frontend/nextjs.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Next.js
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Next.js is a meta-framework for React.
+
+## Checklist
+
+- Use static exports by setting `output: 'export'`. Tauri doesn't support server-based solutions.
+- Use `internal-ip` for mobile compatibility so you can configure `assetPrefix`. This is required so that the server properly resolves assets.
+- Use `out/` as `distDir` in `tauri.conf.json`.
+
+## Example Configuration
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+npm install --save-dev internal-ip
+```
+
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "npm run dev",
+		"beforeBuildCommand": "npm run build",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+
+</TabItem>
+<TabItem label="yarn">
+
+```sh
+yarn add -D internal-ip
+```
+
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "yarn dev",
+		"beforeBuildCommand": "yarn generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+pnpm add -D internal-ip
+```
+
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "pnpm dev",
+		"beforeBuildCommand": "pnpm generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+
+</TabItem>
+</Tabs>
+
+```ts
+// next.conf.ts
+const isProd = process.env.NODE_ENV === 'production';
+module.exports = async (phase, { defaultConfig }) => {
+	let internalHost = null;
+	// In dev mode we use the internal-ip to serve the assets
+	if (!isProd) {
+		const { internalIpV4 } = await import('internal-ip');
+		internalHost = await internalIpV4();
+	}
+	const nextConfig = {
+		// Ensure Next.js uses SSG instead of SSR
+		output: 'export',
+		// Helps you identify issues
+		reactStrictMode: true,
+		// Faster minification
+		swcMinify: true,
```

Comment:
```suggestion
```
This is true as of v13 https://nextjs.org/docs/pages/building-your-application/upgrading/version-13#v13-summary